test/application/team-validator.js

```diff
@@ -34,6 +34,22 @@ describe('Team Validator features', function () {
 			});
 		});
 
+		it('should reject non-existent natures', function (done) {
+			let packedTeam = "|pikachu|eviolite||thunderbolt|nonexistentNature||||||";
+			TeamValidator('customgame').prepTeam(packedTeam).then(result => {
```

Comment:
Anyway, that's a good start, but there's no need to use the async `done` now that the test itself is synchronous.